nemo_skills/pipeline/start_server.py

```diff
@@ -51,7 +51,12 @@
     args = parser.parse_args()
 
     cluster_config = get_cluster_config(args.cluster, args.config_dir)
-    check_if_mounted(cluster_config, args.model)
+
+    if args.server_type != "vllm":
```

Comment:
Let's move this logic inside `get_server_command` so that it automatically applies anywhere server is used. Please also remove the mount check for args.model from generate.py, eval.py, check_contamination.py and llm_math_judge.py (we should unify the common code in those scripts, but not done yet)

Reply:
Done